src/content/docs/release-notes/logs-release-notes/logs-21-12-13.mdx

```diff
@@ -0,0 +1,13 @@
+---
+subject: Logs
+releaseDate: '2021-12-13'
+version: '211213'
+---
+
+### Bug fixes
+
+The log forwarding functionality from [Amazon Kinesis Data Firehose](/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose) was prepending the `message` and `timestamp` attributes with `logEvent`. For example, you might see `logEvent.message` and `logEvent.timestamp` in the UI. This prevented default JSON parsing and also applied the ingestion time rather than the timestamp in the logs. This bug has been fixed.
```

Comment:
Hi @barbnewrelic  , could we mention that our solution was prepending (in `some` cases) customers attributes with `logEvent`? It was not limited to `message` and `timestamp`, but they are good examples.